### ISSUE TITLE
val_loss and val_PSNR stay 0 all the time

### ISSUE THREAD

**Issue by weiwei107** [keras-team/keras#14198]
Hi, I am trying to run this project [https://github.com/yu4u/noise2noise](url). "val_loss" and "val_PSNR" are stuck at 0 all the time, as shown below. Could you please give me some advice on this? 

I am using Keras 2.4.3, Tensorflow 2.3.0, Python 3.8.3. 

Epoch 2/60
2/2 [==============================] - ETA: 0s - loss: 105.5667 - PSNR: 7.5550
Epoch 00002: val_PSNR did not improve from 0.00000
2/2 [==============================] - 5s 2s/step - loss: 105.5667 - PSNR: 7.5550 - val_loss: 0.0000e+00 - val_PSNR: 0.0000e+00

**Comment by weiwei107** [comment]
It seems related to the versions. I changed to python3.6, keras2.2.0, tensorflow1.9.0, and the problem solved.
